src/ensembl/stories/shared-components/instant-download/InstantDownloadTranscript.stories.tsx

```diff
@@ -34,6 +34,10 @@ const buildProteinCodingGene = () => {
         label: 'Protein coding',
         value: 'protein_coding',
         definition: 'Protein coding'
+      },
+      name: {
+        accession_id: 'HGNC:1097',
+        url: 'https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:1097'
```

Comment:
Why does InstantDownloadTranscript need to know about gene name metadata?

Reply:
As discussed in code review, we dont need gene metadata at all, so removed the field completely. I have pushed the changes. Please review. Thanks